### PR TITLE
Tidy styling of “Change the Reclaim Policy of a PersistentVolume

### PR DESCRIPTION
**Pull request by npu21**
https://github.com/kubernetes/website/issues/22781

### CODE REVIEW

File: content/ko/docs/tasks/administer-cluster/change-pv-reclaim-policy.md [hunk 1]
```diff
@@ -19,15 +19,15 @@ content_type: task
 
 ## 왜 퍼시스턴트볼륨 반환 정책을 변경하는가?
 
-`PersistentVolumes` 은 "Retain(보존)", "Recycle(재활용)", "Delete(삭제)" 를 포함한 
-다양한 반환 정책을 갖는다. 동적으로 프로비저닝 된 `PersistentVolumes` 의 경우 
-기본 반환 정책은 "Delete" 이다. 이는 사용자가 해당 `PersistentVolumeClaim` 을 삭제하면, 
+PersistentVolumes 은 "Retain(보존)", "Recycle(재활용)", "Delete(삭제)" 를 포함한 
+다양한 반환 정책을 갖는다. 동적으로 프로비저닝 된 PersistentVolumes 의 경우 
+기본 반환 정책은 "Delete" 이다. 이는 사용자가 해당 PersistentVolumeClaim 을 삭제하면,
```
Comment (seokho-son) [inline content/ko/docs/tasks/administer-cluster/change-pv-reclaim-policy.md:24]:
```suggestion
퍼시스턴트볼륨은 "Retain(보존)", "Recycle(재활용)", "Delete(삭제)" 를 포함한 
다양한 반환 정책을 갖는다. 동적으로 프로비저닝 된 퍼시스턴트볼륨의 경우 
기본 반환 정책은 "Delete" 이다. 이는 사용자가 해당 퍼시스턴트볼륨클레임(PersistentVolumeClaim)을 삭제하면, 
```

File: content/ko/docs/tasks/administer-cluster/change-pv-reclaim-policy.md [hunk 2]
```diff
@@ -19,15 +19,15 @@ content_type: task
...
+기본 반환 정책은 "Delete" 이다. 이는 사용자가 해당 PersistentVolumeClaim 을 삭제하면, 
 동적으로 프로비저닝 된 볼륨이 자동적으로 삭제됨을 의미한다.
 볼륨에 중요한 데이터가 포함된 경우, 이러한 자동 삭제는 부적절 할 수 있다.
 이 경우에는, "Retain" 정책을 사용하는 것이 더 적합하다.
-"Retain" 정책에서, 사용자가 `PersistentVolumeClaim` 을 삭제할 경우 해당하는
-`PersistentVolume` 은 삭제되지 않는다. 
-대신, `Released` 단계로 이동되어, 모든 데이터를 수동으로 복구할 수 있다.
+"Retain" 정책에서, 사용자가 PersistentVolumeClaim 을 삭제할 경우 해당하는
+PersistentVolume 은 삭제되지 않는다. 
+대신, Released 단계로 이동되어, 모든 데이터를 수동으로 복구할 수 있다.
```
Comment (seokho-son) [inline content/ko/docs/tasks/administer-cluster/change-pv-reclaim-policy.md:30]:
```suggestion
"Retain" 정책에서, 사용자가 퍼시스턴트볼륨클레임을 삭제할 경우 해당하는
퍼시스턴트볼륨은 삭제되지 않는다. 
대신, Released(릴리스) 단계로 이동되어, 모든 데이터를 수동으로 복구할 수 있다.
```

**Comment by seokho-son** [comment]
ping ~ @npu21

**Comment by gochist** [comment]
This is duplicated with #23013

**Comment by seokho-son** [comment]
Hi @npu21,

It seems this PR is duplicated with #23013.
Korean localization team maintains outdated Korean contents because of the latest En docs.
The issue that this PR resolves is under a management by PR #23013. 
So, let us close this PR. Thank you for your contribution !

**Pull request closed without merging**